src/util/debug.c

```diff
@@ -53,6 +51,7 @@ enum sss_logger_t sss_logger = STDERR_LOGGER;
 const char *debug_log_file = "sssd";
 FILE *_sss_debug_file;
 uint64_t debug_chain_id;
+const char *debug_chain_id_fmt = "BUG%lu";
```

Comment:
Could you please add a comment like "Default value says 'BUG' because this is just a precautionary measure and it is expected value is always set explicitly by every SSSD process. 'BUG' should make any potential oversight more prominent in the logs."

Reply:
Done